drive/drive.go

```diff
@@ -51,7 +51,8 @@ func (ud *UserDrive) PublicSpace(ctx context.Context, api fxiface.CoreAPI) (*Dri
 		Api:       api,
 		SpaceType: PublicDriveSpaceType,
 		RootCid:   ud.PublicSpaceCid,
-		RootDir:   rootDir.(files.Directory)}}, err
+		RootDir:   rootDir.(files.Directory),
+		Drive:     ud}}, err
```

Comment:
Does this branch need a rebase?

Reply:
@masih yes, it is branched from [Drive space fix](https://github.com/functionland/go-fula/pull/67)